src/components/Inputs/CustomLookup.js

```diff
@@ -73,10 +73,22 @@ const CustomLookup = ({
               return options.filter(option => option)
             }
           }}
-          getOptionLabel={option =>
-            typeof option === 'object' ? `${option[valueField] ? option[valueField] : ''}` : option
-          }
-          isOptionEqualToValue={(option, value) => (value ? option[valueField] === value[valueField] : '')}
+          getOptionLabel={option => {
+            if (typeof valueField == 'object') {
+              const text = valueField
+                .map(header => option[header] && option[header]?.toString())
+                ?.filter(item => item)
+                ?.join(' ')
```

Comment:
remove empty string

Reply:
we need the space between appearing fields